build.xml

```diff
@@ -29,6 +29,12 @@
         </exec>
     </target>
 
+    <target name="create-index" depends="prepare" description="Create index">
```

Comment:
Nejde o lokalne pustanie testov, ide o Jenkins. Ked updatnes v syrupe mapping, tak sa nema ako na do ES s devel prefixom ako dostat - jedine ako to spustis manualne. Na testingu sa to spravi same, lebo tam sa to riesi deploy scriptom.